packages/backend/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "yeoman-ui",
   "displayName": "Application Wizard",
-  "version": "1.10.8",
+  "version": "1.10.9",
```

Comment:
yeoman-ui is a mono-repo where versions are bumped automatically by lerna, please, don't bump version manually